src/macaron/config/defaults.ini

```diff
@@ -318,6 +318,61 @@ deploy_arg =
 github_actions =
     docker/build-push-action
 
+# This is the spec for trusted NPM build tool usages.
+[builder.npm]
+entry_conf =
+    .npmrc
+build_configs =
+    package.json
+package_lock =
+    package-lock.json
+builder =
+    npm
+build_arg =
+    build
```

Comment:
I can have a look into what is commonly used to get a general view of it all, do you think it would be fair to do say a wildcard looking for say *build or similar? So as to try and capture all the different cases. Or should the field just be left blank to trigger the logic in the build service check for no args https://github.com/oracle/macaron/blob/0f426982837708539fb64b76001b823b79596b0e/src/macaron/slsa_analyzer/checks/build_service_check.py#L110

Reply:
I agree with @tromai that `npm build` should not be used here and `npm run build` is probably more relevant. Note that we actually parse the bash commands (not running regex), So, you need to make sure both arguments are passed to `npm` in the check.

On the other hand, the usefulness of [build_service_check.py](https://github.com/oracle/macaron/blob/0f426982837708539fb64b76001b823b79596b0e/src/macaron/slsa_analyzer/checks/build_service_check.py) is not clear anymore because this requirement is removed in SLSA v1.0. As part of transitioning to SLSA v1.0 we will remove this check. So, I think we can skip this check all together.

Reply:
I think you have decided to put it as empty. I think that looks good to me. I will resolve this conversation.